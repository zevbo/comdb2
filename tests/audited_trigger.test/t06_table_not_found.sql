CREATE LUA AUDITED TRIGGER foobar ON (TABLE bx1155 FOR INSERT) 

SELECT "done"