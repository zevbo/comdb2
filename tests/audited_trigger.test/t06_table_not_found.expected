("done"='done')
[SELECT "done"] rc 0
